simulation-ros/src/turtlebot2i/turtlebot2i_description/v-rep_model/NewScriptForDockStation.lua

```diff
@@ -0,0 +1,47 @@
+--## NOTE: This lua file should be inside the model file!
```

Comment:
This file should be the internal file inside `Model.ttt` but I think it is better that @KRaizer or @albertohata  to change them, because the changes in `Model.ttt`could not be easily detected.